boilerplate/app/components/Screen.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { useRef, useState } from "react"
```

Comment:
Screen component has been mostly DRYed up and made more customizable by exposing each sub-component props.